lib/middleman-prismic/commands/prismic.rb

```diff
@@ -1,23 +1,17 @@
+require 'middleman-core/cli'
 require 'yaml'
 require 'fileutils'
 
 module Middleman
   module Cli
-
-    class Prismic < Thor
+    class Prismic < Thor::Group
       # Path where Middleman expects the local data to be stored
-      MIDDLEMAN_LOCAL_DATA_FOLDER = 'data'
+      DATA_DIR = 'data'
```

Comment:
This seems to always be used as `"#{DATA_DIR}/prismic"`—how about including the `/prismic` in the constant?

Also, should this be frozen?